<launch>
    <node pkg="uav_px4_ctrl" type="mission_manger" name="mission_manger" output="screen" >
        <!-- 输入话题（odom 优先） -->
        <param name="odom_topic" value="/localization" />
        <param name="pose_topic" value="/mavros/local_position/pose" />
        <param name="goal_topic" value="/goal_pose" />

        <!-- 到达阈值 -->
        <param name="distance_threshold" value="1.0" />

        <!-- 到达后等待时间 -->
        <param name="wait_time" value="5.0" />

        <!-- 启动延迟时间 -->
        <param name="start_delay" value="3.0" />

        <!-- 位置话题超时时间（秒） -->
        <param name="topic_timeout" value="2.0" />

        <!-- 任务循环选项 -->
        <param name="mission_cycle" value="false" />

        <!-- 路径点 -->
        <param name="waypoints" value="[[52.0, -10.0, 0.6], [52.0, -0.0, 0.6]]"/>
    </node>
</launch>
